python/paddle/optimizer/momentum.py

```diff
@@ -353,6 +485,89 @@ def _append_optimize_op(self, block, param_and_grad):
 
         return momentum_op
 
+    def _append_optimize_multi_tensor_op(self, target_block,
+                                         parameters_and_grads):
+        assert isinstance(target_block, framework.Block)
+
+        self.grad_dict = {'FP32_LODTensor': [], 'FP16_LODTensor': []}
+        self.lr_dict = {'FP32_LODTensor': [], 'FP16_LODTensor': []}
```

Comment:
It is no need to  be `attr` of `self`, temp var is ok

Reply:
Done, tks!